editors/code/src/debug.ts

```diff
@@ -194,7 +194,9 @@ function getCCppDebugConfig(
         args: runnable.args.executableArgs,
         cwd: runnable.args.cwd || runnable.args.workspaceRoot || ".",
         sourceFileMap,
-        env,
+        environment: Object.entries(env).map((entry) => {
+            return { name: entry[0], value: entry[1] };
+        }),
```

Comment:
nit: `return` shouldn't be needed here.
```suggestion
        environment: Object.entries(env).map((entry) => ({
            name: entry[0], value: entry[1]
        })),
```
Not sure what the proper formatting is.

Reply:
You're right, I forgot you could just put parens around the object. Thanks!